redesign.php

```diff
@@ -165,20 +165,244 @@
+                            <div class="panel-text">
+                                <h4>
+                                    John Pfotenhauer:
+                                </h4>
+                                <h5>
+                                    <!-- John Pfotenhauer game -->
+                                </h5>
+                                <p>
+                                    <!-- John Pfotenhauer text -->
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div id="person4" class="app-expanded gridder-content">
+                    <div class="panel">
+                        <div class="panel-body text-left d-flex flex-col flex-lg-row">
+                            <div class="panel-img">
+                                <!-- <img src="/assets/img/gameplay/headlines2.jpg" alt="Headlines and High Water game play"> -->
+                            </div>
+                            <div class="panel-text">
+                                <h4>
+                                    Tamara Thomsen:
+                                </h4>
+                                <h5>
+                                    <!-- Tamara Thomsen game -->
+                                </h5>
+                                <p>
+                                    <!-- Tamara Thomsen text -->
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>     
+
+            <div class="divider"></div>
+
+            <h3>
+                GAMES DESIGNED FOR THE CLASSROOM
+            </h3>
+            <p>
+                Sure, sure, it feels great when kids ask if they can keep playing our games when class is over (the answer is yes!) but our games are designed and tested to work in class.
+            </p>
+            <p>
+                Our games can provide a meaningful experience in a single class period. We create them with input from education standards and our teacher fellowship program.
+            </p>
+            <p>
+                You're never too old to learn through play! We make games for elementary through college and beyond.
+            </p>            
+        
+            <div class="divider"></div>
+
+             <h4>
+                And did we mention they're free?
+             </h4>
+            <p>
+                We don't ever make pay-to-play games. No ads. No grinding. No subscriptions, no logins, no hidden fees,
+                and no microtransactions.
+            </p>
+            <p>
+                Our games are always completely free for the public and available online. Funding comes from public
+                projects, partnerships, and grants.
+            </p>
+            <button class="button small yellow filled">Play our games</button>
```

Comment:
I assume this should just be a link to the `play/` page.

Reply:
done